sdks/python/apache_beam/io/gcp/bigtable_io_test.py

```diff
@@ -0,0 +1,174 @@
+    """ Process beam pipeline using an element.
+
+    :type row_value: dict
+    :param row_value: dict: dict values with row_key and row_content having
+    family, column_id and value of row.
+    """
+    direct_row = row.DirectRow(row_key=row_values["row_key"])
+
+    for row_value in row_values["row_content"]:
+      direct_row.set_cell(row_value["column_family_id"],
+                          row_value["column_id"],
+                          row_value["value"],
+                          datetime.datetime.now())
+
+
+@unittest.skipIf(Client is None, 'GCP Bigtable dependencies are not installed')
+class BigtableIOWriteIT(unittest.TestCase):
+  """ Bigtable Write Connector Test
+
+  """
+  DEFAULT_TABLE_PREFIX = "python-test"
+  instance_id = DEFAULT_TABLE_PREFIX + "-" + str(uuid.uuid4())[:8]
+  cluster_id = DEFAULT_TABLE_PREFIX + "-" + str(uuid.uuid4())[:8]
+  table_id = DEFAULT_TABLE_PREFIX + "-" + str(uuid.uuid4())[:8]
+  number = 500
+  LOCATION_ID = "us-east1-b"
+
+  def setUp(self):
+    try:
+      from google.cloud.bigtable import enums
+      self.STORAGE_TYPE = enums.StorageType.HDD
+    except ImportError:
+      self.STORAGE_TYPE = 2
+
+    self.test_pipeline = TestPipeline(is_integration_test=True)
+    self.runner_name = type(self.test_pipeline.runner).__name__
+    self.project = self.test_pipeline.get_option('project')
+    self.client = Client(project=self.project, admin=True)
+    self._create_instance_table()
+
+  def tearDown(self):
+    instance = self.client.instance(self.instance_id)
+    if instance.exists():
+      instance.delete()
+
+  def test_bigtable_write_python(self):
+    number = self.number
+    config = BigtableWriteConfiguration(self.project, self.instance_id,
+                                        self.table_id)
+    pipeline_args = self.test_pipeline.options_list
+    pipeline_options = PipelineOptions(pipeline_args)
+
+    row_values = self._generate_mutation_data(number)
+
+    with beam.Pipeline(options=pipeline_options) as pipeline:
+      _ = (
+          pipeline
+          | 'Generate Row Values' >> beam.Create(row_values)
+          | 'Generate Direct Rows' >> beam.ParDo(GenerateDirectRows())
+          | 'Write to BT' >> beam.ParDo(WriteToBigtable(config)))
+
+      result = pipeline.run()
+      result.wait_until_finish()
+
+      assert result.state == PipelineState.DONE
+
+      if not hasattr(result, 'has_job') or result.has_job:
+        read_filter = MetricsFilter().with_name('Written Row')
+        query_result = result.metrics().query(read_filter)
+        if query_result['counters']:
+          read_counter = query_result['counters'][0]
+
+          logging.info('Number of Rows: %d', read_counter.committed)
+          assert read_counter.committed == number
+
+  def _create_instance_table(self):
+    """ Prepare all necesary for the test
+    """
+    self.instance = self.client.instance(self.instance_id)
+
+    serve_nodes = 3
+    cluster = self.instance.cluster(self.cluster_id,
```

Comment:
Please create a dev instance, which only has 1 server node.  There's no need for a large instance.